src/app/components/modal/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React, {memo, PropsWithChildren} from 'react';
+import {StyleSheet, View, ViewStyle} from 'react-native';
+
+import ReactNativeModal from 'react-native-modal';
+
+type Props = PropsWithChildren & {
+  style?: ViewStyle | undefined;
+  contentStyle?: ViewStyle | undefined;
```

Comment:
if `style` and `contentStyle` are optional props then I think no need to add undefined

Reply:
Also, can we add the Modal library remaining props importing from library?